### PR TITLE
[TBCM-181] Alert Box for New Plan Creation

### PR DESCRIPTION
**Pull request by kushal-arora-fw**

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -2140,6 +2140,7 @@ __metadata:
     dayjs: 1.11.0
     eslint: 8.0.1
     jest: 26.6.0
+    lodash: 4.17.21
```
Comment (dbayly-freshworks) [inline yarn.lock:2143]: Do you need lodash imported here? It's already installed in common

File: packages/common/src/ro/planningSession.ro.ts [hunk 2]
```diff
@@ -9,12 +38,33 @@ export class PlanningSessionRO {
...
+  @Expose()
+  bundles!: PlanningSessionBundleRO[];
+
   constructor(data: any) {
     Object.assign(this, data);
+
+    /** Bundles */
+    const bundles: PlanningSessionBundleRO[] = [];
+    if (data?.careActivity?.length > 0) {
+      data.careActivity.forEach((ca: any) => {
```
Comment (dbayly-freshworks) [inline packages/common/src/ro/planningSession.ro.ts:55]: Can you add a type here?

Comment (kushal-arora-fw) [reply packages/common/src/ro/planningSession.ro.ts:55]: No - Learned from experience that importing Entities to "common" from "api" leads to RunTime errors. It doesn't translate to @tbcm/common instead the complete non-resolvable path.

**Review by dbayly-freshworks — APPROVED** [review]
This looks ok to me

**Pull request merged**